cfgov/unprocessed/apps/mmt-cost-of-credit-tool/js/components/scroll-to-top.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
```

Comment:
This is another shared React module that both apps are currently using. Just cataloging them all so that we can figure out a solution. Don't do anything to it right now.

Reply:
yes, ok.